roles/multiqc/templates/multiqc_config.yml.j2

```diff
@@ -1,4 +1,4 @@
 ---
 no_version_check: True
 push_statusdb: False
-disable_ngi: False
+"disable_ngi: {{ item.ngi_hooks }}"
```

Comment:
Remove the "" if you do not want them to exist in the outputed config file. They are not parsed/required by jira in this context.